scripts/qa_crowbarsetup.sh

```diff
@@ -4289,7 +4289,7 @@ function oncontroller_testpreupgrade
 function oncontroller_testpostupgrade
 {
     # retrieve the ping results
-    local fips=$(openstack floating ip list -f value -c "Floating IP Address")
+    local fips=$(openstack ip floating list -f value -c IP)
```

Comment:
This may require a guard, since it's 'ip floating' for Cloud 6, and 'floating ip' for Cloud 7.